posts/2021-11-26-jakarta-ee-9.1.adoc

```diff
@@ -0,0 +1,347 @@
+---
+layout: post
+title: "Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12 - OpenLiberty.io
+seo-description: Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers! Several significant bug fixes are also part of this release.
+blog_description: Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers! Several significant bug fixes are also part of this release.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers! Several significant bug fixes are also part of this release.
+ 
+In link:{url-about}[Open Liberty] 21.0.0.12:
+
+* <<jee_9.1, Jakarta EE 9.1 support>>
+* <<response_headers, Configurable Response Headers>>
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A210012+label%3A%22release+bug%22[21.0.0.12].
+
+
+[#run]
+== Run your apps using 21.0.0.12
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>21.0.0.12</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[21.0.0.12,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+[#jee_9.1]
+== Jakarta EE 9.1 support
+
```

Comment:
Add the start of this section, can you add the Jakarta EE logo from https://www.eclipse.org/org/artwork/zip_files/jakartaee_color.zip
![image](https://user-images.githubusercontent.com/3076261/143480361-e69a0c47-3034-48d2-8b39-5ec05f628092.png)